--- bootloaderconfig.h	2012-05-13 11:46:28.000000000 +0200
+++ bootloaderconfig.h.new	2012-07-03 11:13:02.025972446 +0200
@@ -46,7 +46,7 @@
 #define TINY85_HARDWARE_CONFIG_2 	2
 
 /* ---------------------------- Hardware Config ---------------------------- */
-#define HARDWARE_CONFIG 		TINY85_HARDWARE_CONFIG_2
+#define HARDWARE_CONFIG 		TINY85_HARDWARE_CONFIG_1
 
 #define USB_CFG_IOPORTNAME      B
 /* This is the port where the USB bus is connected. When you configure it to
